### DISCUSSION TITLE
Portainer replacement

### DISCUSSION THREAD

**Discussion by HighError** [coollabsio/coolify#1384] [Q&A]
Could this be a replacement for Portainer? I have a server with some running services and I have always missed the ability to run my node.js applications automatically like on Vercel. This system solves this problem, but I would like to know if it can be considered as a replacement/alternative for Portainer. And if so, is it possible to migrate?

Here are some services I use:
1) Postgresql
2) Telegram bot is written in Python (wrapped in docker)
3) Lavalink
4) ~~Cronicle (server for running Cron tasks) (in my case, used to ping an address on one of my projects every day)~~ n8n
5) Nginx Proxy Manager (for proxy ip from docker to my domains)

It would be great to move these services to Coolify and replace the docker image of the python bot with auto deploy using GitHub push/pull requests

**Comment by andyzukunft** [comment]
I would say Portainer has overlapping use cases. Coolify is more complex but it also comes with a lot of more features. If you want to do most of your stuff by hand then Portainer surely is the better solution. If you want to rely on the system to help and provide additional features (databases, log draining, domain / proxy handling) than I would not use Portainer.

**Reply by HighError** [reply]
I understand this, I am more interested in the process of transferring from one system to another. Is it difficult to do? For example, if I uninstall Portainer and install Coolify, will it pick up my docker containers built before that in Portainer?

From what I understand, I will need to migrate my postgresql database to coolify instead of using a docker container?
And can I continue to use NginxProxyManager?

**Reply by andyzukunft** [reply]
I am new to the system myself so I can't say for sure.

HOWEVER what I have learned is that Coolify will not delete your Containers or storage data however it will not display them within Coolify. The application uses a tagging system to identify the relevant containers - I believe.

In my opinion it is not recommendable to drive a "hybrid" installation as this would only increase the administration effort.
